pyrokinetics/cgyro.py

```diff
@@ -702,7 +702,7 @@ def load_fields(self, pyro):
                     )
 
                     complex_field = (
-                        field_data[0, :, :, :, :] + 1j * field_data[1, :, :, :, :]
```

Comment:
Just to check I understand, is this sign change coming from essentially undoing `exp(-i*omega*t)` -> `exp(i*omega*t)` ? Do we need to worry about relative sign of exb shear as well?

Reply:
Is it worth a comment to explain the sign, which otherwise looks a little surprising.

Reply:
Yep that's basically what I've done. Can add a comment clarifying this

Reply:
Not sure about ExB, but that's not implemented in pyrokinetics yet so will keep in mind?